### ISSUE TITLE
BCML graphic packs is missing

### ISSUE THREAD

**Issue by sogata-aw** [NiceneNerd/BCML#364]
So I've installed BCML but in the graphic packs in Cemu, There's no one named BCML to activate.

**Comment by GingerAvalanche** [comment]
This needs more info. Version? Windows/Linux? Mods used?

Assuming you're on Windows, open the start menu, type `%LOCALAPPDATA%\bcml` and hit enter. In that folder, you should see `settings.json` - upload that here. please.

Try running the program with the console command `bcml-debug`, installing mods, and then when it says mod installation has been completed, pasting, here, last two dozen or so lines from the console.

The problem could be caused by a lot of things, but all of that will probably be enough to tell us what's going on.

**Comment by sogata-aw** [comment]
I'm on the version 1.26.2f of Cemu, on the 3.9.9 of BCML and I'm using Windows 10.
For the mods I used:
-Linkle mod
-Linkle Armor Icon Pack
-Linkle Soldier Gear Redone
-Linkle DLC Helmets
-Linkle Mod Title Screen

this is the content of my settings.json :
{
  "cemu_dir": "C:\\Program Files (x86)\\cemu_1.26.2",
  "game_dir": "C:\\Program Files (x86)\\cemu_1.26.2\\mlc01\\usr\\title\\00050000\\101c9500\\content",
  "game_dir_nx": "",
  "update_dir": "C:\\Program Files (x86)\\cemu_1.26.2\\mlc01\\usr\\title\\0005000e\\101c9500\\content",
  "dlc_dir": "C:\\Program Files (x86)\\cemu_1.26.2\\mlc01\\usr\\title\\0005000c\\101c9500\\content\\0010",
  "dlc_dir_nx": "",
  "store_dir": "C:/Program Files (x86)/cemu_1.26.2/graphicPacks/downloadedGraphicPacks/BreathOfTheWild/Mods/BCML",
  "export_dir": "",
  "export_dir_nx": "",
  "load_reverse": false,
  "site_meta": "",
  "no_guess": false,
  "lang": "EUfr",
  "no_cemu": true,
  "wiiu": true,
  "no_hardlinks": false,
  "force_7z": false,
  "suppress_update": false,
  "loaded": true,
  "nsfw": false,
  "changelog": true,
  "strip_gfx": false,
  "auto_gb": true,
  "show_gb": false,
  "dark_theme": false,
  "last_version": "3.9.9"
}

for the debug (I've done it in multiple times due to some crashes):
Opening mod...
Identified mod: Linkle DLC Helmets
Loading mod logs...
Assigned mod priority of 100
Moving mod to C:\Program Files (x86)\cemu_1.26.2\graphicPacks\downloadedGraphicPacks\BreathOfTheWild\Mods\BCML\mods\0100_LinkleDLCHelmets...
Opening mod...
Identified mod: Linkle Soldier Gear Redone
Loading mod logs...
Assigned mod priority of 101
Moving mod to C:\Program Files (x86)\cemu_1.26.2\graphicPacks\downloadedGraphicPacks\BreathOfTheWild\Mods\BCML\mods\0101_LinkleSoldierGearRedone...
Opening mod...
Looks like an older mod, let's upgrade it...
Converting meta file...
Upgrading old logs...
Upgrading RSTB log...
Finding modified files...
Found 889 modified files...
Found 0 modified files in SARCs...
Upgrading text logs...
Identified mod: Linkle Armor Icon Pack
Loading mod logs...
Assigned mod priority of 102
Moving mod to C:\Program Files (x86)\cemu_1.26.2\graphicPacks\downloadedGraphicPacks\BreathOfTheWild\Mods\BCML\mods\0102_LinkleArmorIconPack...
VERBOSEInstalled 1 mods
Installed 1 mods
Cleansing old merges...
Refreshing merged mods...
Loading modded SARC list...
No SARC merging necessary
VERBOSEPackMerger.perform_merge took 0.0020005 seconds
Loading deep merge logs...
No deep merge needed
VERBOSEDeepMerger.perform_merge took 0.0019996 seconds
Loading drop table edits...
No drop table merging necessary
VERBOSEDropMerger.perform_merge took 0.0019995 seconds
Loading deep merge logs...
No deep merge needed
VERBOSEShopMerger.perform_merge took 0.0019989 seconds
Loading AS list merge logs...
No AS list merge needed
VERBOSEASListMerger.perform_merge took 0.003001 seconds
Loading text mods...
No text merge necessary
VERBOSETextsMerger.perform_merge took 0.0029989 seconds
Loading modded actor info...
No actor info merging necessary.
VERBOSEActorInfoMerger.perform_merge took 0.0020001 seconds
VERBOSEDungeonStaticMerger.perform_merge took 0.0019995 seconds
Loading map mods...
No map merge necessary
VERBOSEMapMerger.perform_merge took 0.0030003 seconds
VERBOSEMainfieldStaticMerger.perform_merge took 0.002999 seconds
No gamedata merging necessary.
VERBOSEGameDataMerger.perform_merge took 0.0019987 seconds
No savedata merging necessary.
VERBOSESaveDataMerger.perform_merge took 0.0019998 seconds
Loading event info mods...
No event info merging necessary
VERBOSEEventInfoMerger.perform_merge took 0.002001 seconds
Loading area data mods...
No area data merging necessary
VERBOSEAreaDataMerger.perform_merge took 0.0030002 seconds
Loading status effect mods...
No status effect merging necessary...
VERBOSEStatusEffectMerger.perform_merge took 0.0020001 seconds
VERBOSEResidentsMerger.perform_merge took 0.0020007 seconds
Loading quest mods...
No quest merging necessary
VERBOSEQuestMerger.perform_merge took 0.0019985 seconds
VERBOSERstbMerger.perform_merge took 0.1300019 seconds
Install complete
Exporting merged mod pack...
VERBOSErefresh_master_export took 0.0010003 seconds
VERBOSE[
  {
    "name": "Linkle DLC Helmets",
    "date": "06/25/2022 3:15 PM",
    "priority": 100,
    "path": "C:\\Program Files (x86)\\cemu_1.26.2\\graphicPacks\\downloadedGraphicPacks\\BreathOfTheWild\\Mods\\BCML\\mods\\0100_LinkleDLCHelmets",
    "disabled": false,
    "id": "TGlua2xlIERMQyBIZWxtZXRzPT0xLjAuMA=="
  },
  {
    "name": "Linkle Soldier Gear Redone",
    "date": "06/25/2022 3:15 PM",
    "priority": 101,
    "path": "C:\\Program Files (x86)\\cemu_1.26.2\\graphicPacks\\downloadedGraphicPacks\\BreathOfTheWild\\Mods\\BCML\\mods\\0101_LinkleSoldierGearRedone",
    "disabled": false,
    "id": "TGlua2xlIFNvbGRpZXIgR2VhciBSZWRvbmU9PTEuMC4w"
  },
  {
    "name": "Linkle Armor Icon Pack",
    "date": "06/25/2022 3:17 PM",
    "priority": 102,
    "path": "C:\\Program Files (x86)\\cemu_1.26.2\\graphicPacks\\downloadedGraphicPacks\\BreathOfTheWild\\Mods\\BCML\\mods\\0102_LinkleArmorIconPack",
    "disabled": false,
    "id": "TGlua2xlIEFybW9yIEljb24gUGFjaz09MS4wLjA="
  }
]
VERBOSEBcmlMod(name="Linkle Armor Icon Pack", path="C:/Program Files (x86)/cemu_1.26.2/graphicPacks/downloadedGraphicPacks/BreathOfTheWild/Mods/BCML/mods/0102_LinkleArmorIconPack", priority=102)

And for the Linkle mod and the Linkle Mod Title Screen it crashes after this:

Linkle Mod:

Opening mod...
Identified mod: Linkle Mod
Loading mod logs...
Assigned mod priority of 103
Moving mod to C:\Program Files (x86)\cemu_1.26.2\graphicPacks\downloadedGraphicPacks\BreathOfTheWild\Mods\BCML\mods\0103_LinkleMod...
VERBOSEInstalled 1 mods
Installed 1 mods
Cleansing old merges...
Refreshing merged mods...
Loading modded SARC list...
Merging 16 SARC files...

Linkle Mod Title Screen:

VERBOSE{
  "mods": [
    "C:\\Users\\Sogata\\Downloads\\linkle_mod_title_screen_pack.bnp"
  ],
  "options": {
    "disable": [],
    "options": {}
  },
  "selects": null
}
Opening mod...
Looks like an older mod, let's upgrade it...
Converting meta file...
Upgrading old logs...
Upgrading pack log...
Upgrading RSTB log...
Finding modified files...
Found 1 modified files...
